game/scripts/vscripts/gamemode.lua

```diff
@@ -140,6 +140,7 @@ function GameMode:OnGameInProgress()
   DebugPrint("[BAREBONES] The game has officially begun")
 
   -- initialize modules
+  InitModule(FilterManager)
```

Comment:
This should get initialized earlier, like in [`InitGameMode()`](https://github.com/Trildar/oaa/blob/b005493ff18f02490e4cc6702018ef41a3b7f3da/game/scripts/vscripts/gamemode.lua#L168).